benchmarks/src/pages/RacPage.tsx

```
@@ -0,0 +1,118 @@
+import { useEffect, useMemo, useRef } from 'react'
+import {
+  ListBox,
+  ListBoxItem,
+  ListLayout,
+  Virtualizer,
+  type ListLayoutOptions,
```

ESLint

[error] Member 'ListLayoutOptions' of the import declaration should be sorted alphabetically.

(sort-imports)